go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/robfig/cron v1.2.0 // indirect
 	github.com/sensu/sensu-go/api/core/v2 v2.3.0
-	github.com/sensu/sensu-plugin-sdk v0.13.1
+	github.com/sensu/sensu-plugin-sdk v0.14.1-0.20210910005325-09930cf50ae7
```

Comment:
What's up with the sensu-plugin-sdk tag here?

Reply:
I need to make a small change to the sdk as the underlying output leakage was coming from there.  I'll merge the now approved sdk change and cut a release and normalize the dep.

Reply:
okay the sdk dep has been updated to v0.14.1  instead of the specific commit with the fix.